### ISSUE TITLE
Wrong path in Helm Chart

### ISSUE THREAD

**Issue by mruncleleon** [nucleuscloud/neosync#1405]
I did clone the repository and tried to run helm template but this path is wrong:

https://github.com/nucleuscloud/neosync/blob/cea4e47a7aefecc9eabefb8d734dd6cdd323602e/charts/neosync/Chart.yaml#L26

And when I change to `/frontend/apps/web/charts/app/`  it appears a error message about line 124 in deployment.yaml at `/frontend/apps/web/charts/app/templates/` were is commented about Tilt.

Is there a chart repository for Neosync? Or we need to clone these folders locally?

**Comment by nickzelei** [comment]
Hi there,
You're right, that chart path is incorrect.
When we publish this chart, we replace the repository with the built images for each sub-chart.

All of our charts are pre-built and available in the ghcr.io registry.

Here is a link to all of the available artifacts: https://github.com/orgs/nucleuscloud/packages?repo_name=neosync

here is the github link to the helm chart you are referencing: https://github.com/nucleuscloud/neosync/pkgs/container/neosync%2Fhelm%2Fneosync

The chart itself can be pulled from ghcr.io via: `ghcr.io/nucleuscloud/neosync/helm/neosync:0.3.39`
We don't host a traditional helm chart repository, but you can use OCI images in helm to inflate and install. 

Here is an example of how we do this for our environments with helmfile:

```
repositories:
  - name: neosync
    url: ghcr.io/nucleuscloud/neosync/helm
    oci: true

---
releases:
  - name: neosync
    chart: neosync/neosync
    version: '0.0.39'
    namespace: neosync
    createNamespace: true

    values:
```

Regardless, still worth fixing it for local testing and thanks for calling it out.

**Comment by nickzelei** [comment]
Fixed via #1451

**Comment by nickzelei** [comment]
> Thank you, Nick. Sorry for late response.
> 
> I did find another line that doesnt work because `backend` was renamed to `api`.
> 
> https://github.com/nucleuscloud/neosync/blob/main/backend/charts/api/templates/deployment.yaml#L62
> 
> I didnt find values.yaml for neosync, only for worker, api and app.

Nice catch! I've fixed this as well. I checked our deployment scripts and we are always providing the url, so never caught the fallback.
